content/docs/0.8.1/advanced-resources/deploy/airgap.md

```diff
@@ -268,68 +261,63 @@ weight: 2
+    "Dropping Longhorn node longhorn-system/**NODE_NAME** out of the queue: fail to sync node for longhorn-system/**NODE_NAME**: 
+    InstanceManager.longhorn.io \"instance-manager-e-605e9473\" is invalid: metadata.labels: Invalid value:
+    \"**PRIVATE_REGISTRY_URL**-**PREFIX**-longhorn-instance-manager-v1_20200301\": **must be no more than 63 characters**"
 
 
 
 #### Longhorn instance manager: metadata.labels must be no more than 63 characters
 
-Using a long registry URL may cause Longhorn installation error
+Using a long registry URL may cause Longhorn installation error.
+
+Longhorn manager reports errors in the log when this happens:
+
+    "instance-manager-e-xxxxxxxx" is invalid: metadata.labels: Invalid value: "<PRIVATE_REGISTRY_URL>-longhornio-longhorn-instance-manager-v1_20200301": must be no more than 63 characters
 
-Longhorn manager would report errors in the log when this happened:
-```
-"instance-manager-e-xxxxxxxx" is invalid: metadata.labels: Invalid value: "<PRIVATE_REGISTRY_URL>-longhornio-longhorn-instance-manager-v1_20200301": must be no more than 63 characters
-```
+Longhorn instance manager pods have labels with the key `longhorn.io/instance-manager-image` and the value `REGISTRY_URL-USER-IMAGE_NAME-TAG`, e.g.:
+
+    metadata:
+      labels:
+        longhorn.io/component: instance-manager
+        longhorn.io/instance-manager-image: <PRIVATE_REGISTRY_URL>-longhornio-longhorn-instance-manager-v1_20200301
+        longhorn.io/instance-manager-type: engine
+        longhorn.io/node: <NODE_NAME>
+      name: instance-manager-e-XXXXXXXX
 
-Longhorn instance manager pods have labels with key `longhorn.io/instance-manager-image` and value `REGISTRY_URL-USER-IMAGE_NAME-TAG`
 
-e.g  
-```
-metadata:
-  labels:
-    longhorn.io/component: instance-manager
-    longhorn.io/instance-manager-image: <PRIVATE_REGISTRY_URL>-longhornio-longhorn-instance-manager-v1_20200301
-    longhorn.io/instance-manager-type: engine
-    longhorn.io/node: <NODE_NAME>
-  name: instance-manager-e-XXXXXXXX
-```
+It's a known Kubernetes limitation that label values should be no more than 63 characters [here.](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
 
-it's known Kubernetes limitaion that label value should be no more than 63 characters [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
+##### Recommendation for Image Tags
 
-##### Recommendation:
-it's highly recommended **not** to maniplute image tags, specially instance manager image tag e.g `v1_20200301`, since we intentionally used date so it is not assoctiated with Longhorn version.
+It's highly recommended **not** to manipulate image tags, especially instance manager image tags such as `v1_20200301`, because we intentionally used the date to avoid associating it with a Longhorn version.
```

Comment:
Could we change it to "It's highly recommended not to manipulate image tags, especially instance manager image tags such as v1_20200301, because we intentionally **use** the date to avoid associating it with a Longhorn version."

Reply:
updated